### ISSUE TITLE
Improve User Experience

### ISSUE THREAD

**Issue by TurtleP** [wiiu-env/gdbstub_plugin#6]
Currently the gdb stub is useful, but has its drawbacks:

1. It will always wait for the client to attach. This results in confusion if the user is not familiar with the plugin. Initially when it came out I was super excited, but it hung the home menu and then when I asked about it, was told that it will always wait for the gdb client and you have to run your app first and *then* wiiload the stub plugin. It's a bad workflow.

2. In a scenario where the application can't be run first, the stub is completely useless. For example, I was running into crashes with LÖVE Potion quite frequently when first working on the Wii U version. It's not very useful to wiiload the stub and then attach the client from the home menu as I don't believe I'll get the expected information because I'd be debugging the home menu and not my application. Plus, due to my application hanging, I can't load it and then the gdb stub.

## Solution

Ideally, the stub could use some more patches where there is a menu item similar to how it is on Nintendo 3DS with luma3ds.In there you can choose to enable the gdb stub, but it will not wait for the client. You can then run your app and attach the gdb client whenever. It also has another menu item to force debug the next launched application. This is useful for my scenario where a user's app may be hanging on startup and it is difficult to use the gdb stub to even debug the issue, aside from attempt to read the slc storage crash logs.

I have spoken to Gary on Discord about this prior, but I figured an issue should be opened to help get the idea out and hopefully dealt with sooner than later.

**Comment by Clownacy** [comment]
> In a scenario where the application can't be run first, the stub is completely useless.

This is not true: you can load the stub on the Home Menu and then load your homebrew and it will properly 'reattach'. I'm doing it right now to debug my own homebrew.
